[Map] Controller support

Unarelith opened the pull request:
Also includes integration in Player prefab (only local to the scene).

Checklist:
- [x] Hide the cursor on the in-game map
- [x] Display the pins on multiple lines
- [x] Handle grid movement (currently every direction move to the next...)
- [x] Only show the cursor in controller mode (could be shown on selected pin in mouse+kb mode)

Unarelith committed:
[Map] Integrated in-game map into player map plane.

Unarelith committed:
[MapTool] Now using IHandTool.

Unarelith committed:
[Map] Now using Submit and Cancel.

[PlayerActionInputs] Removed most of Map and Stamp action maps.

Mystos commented:
Task linked: [CU-f11va6 Controller support on fullscreen map](https://app.clickup.com/t/f11va6)

Unarelith committed:
[Map] Added a cursor for controller mode.

Unarelith committed:
[Map] Pin placement with controller seems ok.

Unarelith committed:
[Map] Improved pin selection / placement.

Unarelith committed:
[Map] Retagged ApertureMask and FogOfWarCamera.

Unarelith committed:
[Map] Controller support : OK.

- Fixed issues with pin selection in panel
- Added pin deletion on Del/North button

Unarelith committed:
[Map] Cursor is no longer visible on in-game map.

Unarelith committed:
[Map] Pins are now displayed on multiple lines.

Unarelith committed:
[Map] Grid movement on map pin panel.

Unarelith committed:
[Map] Big refactoring.

- Moved UI control to MapUI
- Moved event handling to MapEventHandler
- Moved pin placement/removal code to MapPinManager
- Moved cursor-related code to MapCursor

Unarelith committed:
[Map] Updated MapCanvas prefab.

Unarelith commented:
Ready for review.
